python/psim/cli/config.py

```diff
@@ -0,0 +1,35 @@
+from .. import config
+
+import argparse
+import json
+
+
+def main(args):
+    
+    parser = argparse.ArgumentParser(
+        prog='psim config', description='Process PSim configuration files.'
+    )
+    parser.add_argument(
+        '--modules', '-m', nargs='+', required=False, type=str, help='Set of ' +
+        'modules to process configuration files for. Defaults to all modules ' +
+        'none are specified.'
+    )
+    parser.add_argument(
+        '--output', '-o', type=argparse.FileType('w'), default='-',
+        help='Destination file for the processed configuration file. Defaults' +
```

Comment:
Is this actually defaulting to stdout? I see `-` as the default argument

Reply:
Interesting, it seems to work.

Reply:
Found it online - don't fully understand it lol